keanu-project/src/main/java/io/improbable/keanu/templating/SequenceBuilder.java

```diff
@@ -42,10 +45,39 @@
         Sequence build();
     }
 
-    public static VertexLabel proxyFor(VertexLabel label) {
+    public static VertexLabel proxyLabelFor(VertexLabel label) {
         return label.withExtraNamespace(PROXY_LABEL_MARKER);
     }
 
+    public static DoubleProxyVertex doubleProxyFor(VertexLabel label) {
```

Comment:
@migwellian how would you feel about this method being a non-static method on each `SequenceItem`? This would mean we could actually also call the `sequenceItem.add(proxy)` and remove another step for the developer.

Reply:
Yes, that's a good idea